src/Common/tests/System/Net/Http/Http2LoopbackServer.cs

```diff
@@ -0,0 +1,197 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
```

Comment:
Just wondering if you need all these 'using' statements?  If you use Visual Studio it will grey out the ones you don't need, in which case you can delete those 'using' lines.

Reply:
I was able to remove the majority of these in the latest commit.